CommonTools/ParticleFlow/python/pfCHS_cff.py

```diff
@@ -1,15 +1,10 @@
 import FWCore.ParameterSet.Config as cms
-from CommonTools.ParticleFlow.pfNoPileUpJME_cff import adapt, pfPileUpJME
-from CommonTools.RecoAlgos.sortedPackedPrimaryVertices_cfi import sortedPackedPrimaryVertices
+from CommonTools.ParticleFlow.pfNoPileUpJME_cff import primaryVertexAssociationJME
 
-packedPrimaryVertexAssociationJME = sortedPackedPrimaryVertices.clone(
-  produceSortedVertices = False,
-  producePileUpCollection  = False,
-  produceNoPileUpCollection = False
-)
-adapt(packedPrimaryVertexAssociationJME)
-
-from CommonTools.ParticleFlow.pfNoPileUpPacked_cfi import pfNoPileUpPacked as _pfNoPileUpPacked
-pfCHS = _pfNoPileUpPacked.clone(
-    vertexAssociationQuality=pfPileUpJME.vertexAssociationQuality
+pfCHS = cms.EDFilter("CandPtrSelector",
+              src = cms.InputTag("packedPFCandidates"),
+              cut = cms.string("fromPV(0)>0"+\
+	       " || (vertexRef().key<="+str(primaryVertexAssociationJME.assignment.NumOfPUVtxsForCharged.value())+" && "+\
+		    "abs(dz(0))<"+str(primaryVertexAssociationJME.assignment.DzCutForChargedFromPUVtxs.value())+")"
```

Comment:
Perhaps a format string can be considered:
```suggestion
              cut = cms.string("fromPV(0)>0 || (vertexRef().key<={} && abs(dz(0))<{})".format(
                  primaryVertexAssociationJME.assignment.NumOfPUVtxsForCharged.value(),
                  primaryVertexAssociationJME.assignment.DzCutForChargedFromPUVtxs.value())
```